Cargo.toml

```
@@ -21,11 +21,11 @@ include = [
 edition = "2021"
 
 [dependencies]
-embedded-hal = "0.2.7"
+embedded-hal = "1.0.0"
 
 [dev-dependencies]
-linux-embedded-hal = "0.3"
-embedded-hal-mock = "0.9"
+linux-embedded-hal = "0.4"
+embedded-hal-mock = {version="0.10.0", features=['eh1']}
```

```suggestion
embedded-hal-mock = {version="0.10.0", default-features=false, features=['eh1']}
```
To avoid the dependency on e-h 0.2.

---

I added the suggestion, updated the changelog, and ran cargo fmt. 

I think it should be good to go!